src/style.css

```diff
@@ -0,0 +1,63 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+* {
+  margin-top: 0;
```

Comment:
there is no default margin for form so this line of code is redundant

Reply:
@amitpatiljc why? I delete it and the top margin moves up which is making me fail the only test case I pass